### DISCUSSION TITLE
Error - ProviderFailToDownloadFile,Microsoft.PowerShell.PackageManagement.Cmdlets.SavePackage - Windows (Install-PowershellAzModules.ps1)

### DISCUSSION THREAD

**Discussion by LokeshNagireddy** [actions/virtual-environments#9903] [General questions]
Hi Team,

After fixing the chocolatey installation, my build got the below error while installing Install-PowershellAzModules.ps1.

![image](https://github.com/actions/runner-images/assets/113025693/f53ec401-fc81-4f97-bac6-989578c9728c)

Can you please look into this issue and solve the same.

i look forward for your response.

**Comment by LokeshNagireddy** [comment]
Hi @erik-bershel , Can you please have a look at this issue.

**Reply by erik-bershel** [reply]
I'll be busy this week. Some of the other maintainers will take a look possibly.

**Comment by LokeshNagireddy** [comment]
Hi @mikhailkoliada , May you please look into this issue, we are behind our build schedule from many days.. if your are busy, can you please assign to any potential collaborator.

i look forward for your response.
